src/providers/twitch/TwitchChannel.cpp

```diff
@@ -408,6 +495,18 @@ void TwitchChannel::sendMessage(const QString &message)
     {
         qCDebug(chatterinoTwitch) << "sent";
         this->lastSentMessage_ = parsedMessage;
+        if (!this->hasHighRateLimit())
+        {
+            auto roomModes = *this->accessRoomModes();
+            if (roomModes.slowMode > 0)
+            {
+                // XXX: this'll (incorrectly) reset the countdown if we still
+                // have time remaining if the user was too eager to send a
```

Comment:
```suggestion
                // have time remaining, if the user was too eager to send a
```
also thought about replacing if with because but 🤷